### DISCUSSION TITLE
tandoor "system" setting error

### DISCUSSION THREAD

**Discussion by redstormsju** [TandoorRecipes/recipes#3409] [Q&A]
Hello all...

So I've installed tandoor on my nas using docker compose (portainer).  I am able to install it and open up the app just fine.  However, when I click on the "system" option in the drop down menu on the top right, I get a 'Server Error (500)' page.  Prior to this I would get another error page that at the very bottom of the page said that to correct the error I needed to set DEBUG to false....below are my environment variables for for this...
environment:
      - SECRET_KEY=#$*#(&*(&FHOISDHF*
      - DB_ENGINE=django.db.backends.postgresql
      - POSTGRES_HOST=jag-tandoor-db
      - POSTGRES_PORT=5432
      - POSTGRES_USER=tandoor
      - POSTGRES_PASSWORD=#$#&*$(&#*&(^
      - POSTGRES_DB=tandoordb
      - DEBUG=0
      - SQL_DEBUG=0
      - ALLOWED_HOSTS=*
      
Not sure how to fix it...any help is appreciated!

**Comment by vabene1111** [comment]
issue in the current version, you can safely ignore that.

**Reply by redstormsju** [reply]
Thank you!.....when i look at the db logs, I am getting this persistent error - 2024-11-23 20:46:27.202 UTC [413] FATAL:  role "root" does not exist.... however, the status of the db is healthy on my docker instance....okay to ignore?
